server/src/main/java/org/apache/uniffle/server/ShuffleServerMetrics.java

```diff
@@ -345,5 +358,33 @@ private static void setUpMetrics() {
     summaryTotalRemoveResourceTime = metricsManager.addSummary(TOTAL_REMOVE_RESOURCE_TIME);
     summaryTotalRemoveResourceByShuffleIdsTime =
         metricsManager.addSummary(TOTAL_REMOVE_RESOURCE_BY_SHUFFLE_IDS_TIME);
+
+    gaugeDataSizeTotalUsage =
+        Gauge.build()
+            .name(TOPN_OF_TOTAL_DATA_SIZE_FOR_APP)
+            .help("top N of total shuffle data size for app level")
+            .labelNames("topn_of_total_data_size_for_app")
```

Comment:
the label name is wrong. If we hope this could be shown to find out the app and sorted number, these labels should be taken.

Reply:
@zuston 

This is the first initialization in setUpMetrics() in ShuffleServerMetrics.java, and to make gaugeDataSizeTotalUsage not be null.

The real metric cals is in incStorageSuccessCounter() in TopNShuffleDataSizeOfAppCalcTask.java.
eg:
`   List<Map.Entry<String, ShuffleTaskInfo>> topNTaskInfo =
        calcTopNTotalDataSizeTaskInfo();
    gaugeTotalDataSize.clear();
    for (Map.Entry<String, ShuffleTaskInfo> taskInfo : topNTaskInfo) {
      gaugeTotalDataSize.labels(taskInfo.getKey()).set(taskInfo.getValue().getTotalDataSize());
    }
`

In order to no make misunderstand， how about I delete the init and add null check ?

Reply:
Or How about code:

      ```
  Gauge.build()
            .name(TOPN_OF_TOTAL_DATA_SIZE_FOR_APP)
            .help("top N of total shuffle data size for app level")
            .labelNames("")
```